pennylane/__init__.py

```diff
@@ -26,6 +26,7 @@
 import pennylane.numpy
 from pennylane.queuing import QueuingManager, apply
 
+import pennylane.capture
 import pennylane.kernels
 import pennylane.math
 import pennylane.operation
```

Comment:
This should sit at least before `pennylane.operation`, but could be moved down a bit if preferred.